backend/src/main/java/io/papermc/hangar/security/configs/SecurityConfig.java

```diff
@@ -93,15 +93,15 @@ public AuthenticationManager authenticationManagerBean() {
     }
 
     @Bean
-    public CorsFilter corsFilter() {
+    public UrlBasedCorsConfigurationSource corsConfigurationSource() {
         final UrlBasedCorsConfigurationSource source = new UrlBasedCorsConfigurationSource();
         final CorsConfiguration config = new CorsConfiguration();
         config.addAllowedOrigin("*");
         config.addAllowedHeader("*");
         config.addAllowedMethod("*");
         source.registerCorsConfiguration("/api/v1/", config);
```

Comment:
```suggestion
        source.registerCorsConfiguration("/api/v1/**", config);
```
this is the actual bug